### DISCUSSION TITLE
The project is really interesting and very useful. I'd like to know if there's a team working on SLAM.

### DISCUSSION THREAD

**Discussion by AnsoATC** [abizovnuralem/go2_ros2_sdk#40] [Q&A] [answered]

**Comment by abizovnuralem** [comment, marked as answer]
Yes. need some time to fix some issues with nav2 lib and map gen in multiple floors

**Comment by MomoHsieh** [comment]
Hi, I would like to ask a few questions following this topic. 
1. From the official sources, I understand that AIR and PRO are not open for further development and without another lidar. Is that navigation can support on AIR or PRO?

2. Currently, you are pulling 3D point cloud data and IMU information back to the computer via WebRTC. Is there an issue with real-time performance when using this setup for mapping and navigation?

3. You plan to use the NAV2 package for navigation and utilize 3D point cloud data for 3D mapping and navigation. However, does this present any problems with the angle coverage (the current field of view is only about 45 degrees)?

Thanks

**Comment by abizovnuralem** [comment]
1. The go2 lidar is enough to make a simple navigation, it produces a good point cloud that you can use to create a map
2. Depends on the wifi signal (use wifi 6), in some cases better to use CycloneDDS via ethernet because the refresh rate is doubled if you compare with webrtc. 
3. Currently, I don't know, I hope it will work fine with large maps.

**Reply by AnsoATC** [reply]
Hi,

I apologize if my questions have already been addressed, but I want to make sure I understand the specifics. I'm interested in purchasing a Unitree Go2 model, but my budget is limited to the AIR model.

Have you tested the current setup with the AIR model? If so, what were the results, and were there any specific challenges or limitations encountered?
Given the constraints with lidar and real-time data transmission, would you recommend any specific configurations or hardware adjustments to optimize navigation on the AIR model?
Thank you for your help!

**Reply by abizovnuralem** [reply]
It works with AIR version, already did some tests. If you are limited on budget, no problem with it. In the other case, it is better to buy EDU version.

**Reply by AnsoATC** [reply]
Thank you very much for your answer. Really pleased with your quick response.
